index.html

```diff
@@ -579,7 +579,7 @@ <h4><dfn>Process pending pointer capture</dfn></h4>
 
                 <section>
                     <h4>Suppressing a pointer event stream</h4>
-                    <p>The user agent MUST <dfn>suppress a pointer event stream</dfn> when it detects that a pointer is unlikely to continue to produce events.  Any of the following scenarios satisfy this condition (there MAY be additional scenarios):</p>
+                    <p>The user agent MUST <dfn>suppress a pointer event stream</dfn> when it detects that the web page is unlikely to continue to receive pointer events with a specific <code>pointerId</code>.  Any of the following scenarios satisfy this condition (there MAY be additional scenarios):</p>
```

Comment:
```suggestion
                    <p>The user agent MUST <dfn>suppress a pointer event stream</dfn> when it detects that the web page is unlikely to continue to receive pointer events with a specific <code>pointerId</code>. Any of the following scenarios satisfy this condition (there MAY be additional scenarios):</p>
```

looks good to me. just removing a stray extra space that was there originally already